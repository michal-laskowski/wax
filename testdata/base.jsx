export function base() {
    return <>
        <a href="test">a link</a>
        <ul>
            {[1, 2].map((li) => (
                <li>{li}</li>
            ))}
        </ul>
        <button onclick="doSomething"></button>
        <div class="class-a"></div>
        
        <script
            src="jquery.js"
            integrity="sha256-123="
            crossorigin="anonymous"
        ></script>

        <svg width="100" height="100" viewBox="0 0 100 100">
            <circle
                cx="50"
                cy="50"
                r="40"
                stroke="green"
                stroke-width="4"
                fill="yellow"
            />
        </svg>
    </>
}
