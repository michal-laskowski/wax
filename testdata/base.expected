<a href="test">a link</a>
<ul><li>1</li><li>2</li></ul>
<button onclick="doSomething"></button>
<div class="class-a"></div>
<script src="jquery.js" integrity="sha256-123=" crossorigin="anonymous"></script>
<svg width="100" height="100" viewBox="0 0 100 100">
    <circle cx="50" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow"></circle>
</svg>